### DISCUSSION TITLE
AW3423DWF Support?

### DISCUSSION THREAD

**Discussion by SubOxyde2010** [T-Troll/alienfx-tools#350] [Q&A]
Hey there! I installed this program, and unfortunately do not have any of the options shown in the beginner's walkthrough. DirectX is installed and my AW3423DWF monitor is up to date, I also have it connected via USB and DisplayPort. I can control the lights using AWCC, but Alien FX tools does not appear to detect anything. I am ultimately just wanting to utilize the ambient functionality of the app to light up the back of the monitor with what's on screen.

Is this monitor currently unsupported, or am I doing something wrong?

![image](https://github.com/T-Troll/alienfx-tools/assets/128816522/ccefa094-bd3e-4d9a-ba8b-83e8b0a5548b)

**Comment by T-Troll** [comment]
It should be supported.
Please run `alienfx-cli probe` from app folder and share me its output (answer "n" for question).

**Reply by dc-IV-AW** [reply]
Did you get an answer on this? I also have the AW3423DWF.

**Reply by T-Troll** [reply]
In process. It uses new encrypted protocol. We are hack it already, but implementation will take time.
So be patient.
